app/scripts/filters/mobile.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+angular.module('openshiftConsole')
+  .filter('isMobileService', function() {
+    return function(serviceInstance) {
+      return _.get(serviceInstance, 'metadata.labels.mobile', {}) === 'enabled';
```

Comment:
we are planning on removing the need to label the service instances soon so we will need another way to do this, we plan on adding the service instance id as a label to the configmap, so perhaps this could change in the future to look for a corrosponding configmap (this is a not to myself) ping @philbrookes